### PR TITLE
Contribution: Education - Translation - Ru - Chapter I - Welcome to Superalgos! - 81-116

### PR DESCRIPTION
**Pull request by maxah**

### CODE REVIEW

File: log.txt [hunk 1]
```diff
@@ -0,0 +1,199 @@
+1202308651 Projects/Superalgos/Schemas/Docs-Tutorials/welcome-to-superalgos-013-tutorial-step-how-are-things-going.json                                  (Luis Fernando Molina 2021-01-08 21:03:04 +0100   1) {
+1202308651 Projects/Superalgos/Schemas/Docs-Tutorials/welcome-to-superalgos-013-tutorial-step-how-are-things-going.json                                  (Luis Fernando Molina 2021-01-08 21:03:04 +0100   2)     "tutorial": "Welcome to Superalgos!",
```
Comment (gitforkgit) [inline log.txt:2]: Is this file meant to be submitted?

Comment (maxah) [reply log.txt:2]: > Is this file meant to be submitted?

I don't know.

Comment (gitforkgit) [reply log.txt:2]: There doesn`t seem to be a path and i can`t find an existing log.txt file. You added it. Have you spoken with the docs Team or julian-molina that you will add this file or did it happen accidentally. If it happened accidentally could you remove it from this Pull request so the pull request can get merged?
Greetings :+1:

Comment (maxah) [reply log.txt:2]: I figured it out. I'll fix it soon.

Comment (maxah) [reply log.txt]: Done.

Comment (gitforkgit) [reply log.txt]: Nice, i will look through the Pull request. This may take a time as it are some files.

**Review by gitforkgit — APPROVED** [review]

**Comment by gitforkgit** [comment]
@Luis-Fernando-Molina can you approve this pull request from the foundations project side?

**Comment by harrellbm** [comment]
@gitforkgit These are all docs pages. It looks like foundations is getting flagged because that is where some of the pages live.  It would be good to mark down which pages those are and see if it makes sense to move them to be housed in a different project

**Pull request merged**